mapper/mapper.go

```diff
@@ -142,6 +143,20 @@ func FeaturedContent(homepageData zebedee.HomepageContent, images map[string]ima
 	return mappedFeaturesContent
 }
 
+// AroundONS takes the homepageContent as returned from the client and returns an array of featured content
+func AroundONS(homepageData zebedee.HomepageContent, images map[string]image.ImageDownload) []model.Feature {
+	var mappedAroundONS []model.Feature
+	for _, fc := range homepageData.AroundONS {
```

Comment:
is it possible for `homepageData.AroundONS` to be nil? If so we should add a check to protect against this if necessary.

Reply:
added check for nil AroundONS property